commit 6575c523d3f30415c7e0a92f1b41ea0d610a206c
Author: Olivier Fourdan <ofourdan@redhat.com>
Date:   Thu Jun 28 18:41:25 2012 +0200

    wacom: do not rotate pad devices
    
    as the rotation property is to be removed on pad devices
    in the driver.
    
    https://bugzilla.gnome.org/show_bug.cgi?id=679077

diff --git a/plugins/wacom/gsd-wacom-manager.c b/plugins/wacom/gsd-wacom-manager.c
index 5edddf2..f7c4610 100644
--- a/plugins/wacom/gsd-wacom-manager.c
+++ b/plugins/wacom/gsd-wacom-manager.c
@@ -646,7 +646,8 @@ wacom_settings_changed (GConfValue     *value,
 	type = gsd_wacom_device_get_device_type (device);
 
 	if (g_str_equal (key, KEY_ROTATION)) {
-		set_rotation (device, gsd_wacom_get_rotation_type_from_name (gconf_value_get_string (value)));
+	        if (type != WACOM_TYPE_PAD)
+		        set_rotation (device, gsd_wacom_get_rotation_type_from_name (gconf_value_get_string (value)));
 	} else if (g_str_equal (key, KEY_TOUCH)) {
 		set_touch (device, gconf_value_get_bool (value));
 	} else if (g_str_equal (key, KEY_TPCBUTTON)) {
